test/dynamo/test_higher_order_ops.py

```diff
@@ -2436,7 +2437,8 @@ def tearDown(self):
                 "Interpreter stack is not empty. Test should have called "
                 "'torch._C._functorch._vmap_decrement_nesting()'"
             )
-            self.fail(msg)
+            warnings.warn(msg)
+            torch._C._functorch._vmap_decrement_nesting()
```

Comment:
Btw, you want to make sure you undo all of the functorch stack. It's possibl that we're in multiple nested vmaps when dynamo decides to break on us